app/src/main/java/net/gsantner/markor/activity/DocumentEditAndViewFragment.java

```diff
@@ -839,28 +841,6 @@ public void webViewJavascriptCallback(final String[] jsArgs) {
         }
     }
 
-    private static boolean fadeInOut(final View in, final View out) {
```

Comment:
This is a common utility - moved to contextutils (probably should create GsViewUtils)